File: mobile-sdk/configurations/cs/show-information.cs
// The sandbox API key passed in 'ApiKeyClientAuthorization' is public.
// Don't submit any personally identifiable information in any requests made with this key.
// Sign in to developer.dojo.tech to create your own private sandbox key and use that instead
// for secure testing.

var paymentIntentsClient = new PaymentIntentsClient(new HttpClient(), new ApiKeyClientAuthorization("sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"));
var result = await paymentIntentsClient.CreatePaymentIntentAsync(new CreatePaymentIntentRequest
{
    Amount = new()
    {
        Value = 4000,
        CurrencyCode = "GBP"
    },
    ItemLines = new List<ItemLine>()
    {
        new() {AmountTotal = new() { Value=1200, CurrencyCode="GBP" }, Caption = "Baseball hat", Id="item 1"},
        new() {AmountTotal = new() { Value=350, CurrencyCode="GBP" }, Caption = "Baseball ball", Id="item 2"},
        new() {AmountTotal = new() { Value=2500, CurrencyCode="GBP" }, Caption = "Baseball bat", Id="item 3"},        
    },
    Config = new()
    {
        Details = new()
        {
            ShowTotal = true
        }
    },
    PaymentMethods = new List<PaymentMethod>() { PaymentMethod.Card , PaymentMethod.Wallet},
    Reference = "Order-0001"
});

File: mobile-sdk/configurations/python/show-information.py
# The sandbox API key passed in 'authorization' is public.
# Don't submit any personally identifiable information in any requests made with this key.
# Sign in to developer.dojo.tech to create your own private sandbox key and use that instead
# for secure testing.

import http.client
import json

conn = http.client.HTTPSConnection("api.dojo.tech")
payload = json.dumps({
    "amount": {
        "value": 4000,
        "currencyCode": "GBP"
    },
    "reference": "Order 245",
    "itemLines": [
        {
          "id": "item 1",
          "name": "Baseball hat",
          "amountTotal": {
                  "value": 1200,
                  "currencyCode": "GBP"
              }
        },
        {
          "id": "item 2",
          "name": "Baseball ball",
          "amountTotal": {
                  "value": 350,
                  "currencyCode": "GBP"
              }
        },
        {
          "id": "item 3",
          "name": "Baseball bat",
          "amountTotal": {
                  "value": 2500,
                  "currencyCode": "GBP"
              }
        }
    ],
})
headers = {
    'Content-Type': "application/json",
    'Version': "2022-04-07",
    'Authorization': "Basic sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"  # <-- Change to your secret key
}
conn.request("POST", "/payment-intents", payload, headers)
res = conn.getresponse()
data = res.read()
print(data.decode("utf-8"))

conn.close()